test/unit_tests/organisms/Footer-spec.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+var BASE_JS_PATH = '../../../cfgov/unprocessed/js/';
+
+var chai = require( 'chai' );
+var expect = chai.expect;
+//var Footer = require( BASE_JS_PATH + 'organisms/Footer' );
```

Comment:
For whatever reason it seems the two specs requiring jQuery get messed up when including the footer organism. I'm removing them since we've agreed we're going to replace them post-launch.